### PR TITLE
Split Cucumber steps a bit further

### PR DESCRIPTION
**Pull request by harrisj**
I have decided against having a large switch statement in steps, so I have unrolled a few of these into separate step definitions

### CODE REVIEW

**Comment by harrisj** [comment]
It looks like there is an issue with timezones. Investigating.

**Comment by jessieay** [comment]
@harrisj looks like the same issue I ran into yesterday evening. If you rebase, should be fixed

File: features/vendor_views_small_biz_auctions.feature [hunk 1]
```diff
@@ -12,7 +12,7 @@ Feature: Vendor views auctions set aside for small businesses only
     And I should see a "Small-business only" status
       
   Scenario: A vendor sees a non small business only auctions on the auction view page
-    Given there is a below the micropurchase threshold auction
+    Given there is an auction below the micropurchase threshold
```
Comment (jessieay) [inline features/vendor_views_small_biz_auctions.feature]: u write good

**Comment by jessieay** [comment]
like this change a lot! Mind rebasing so we are sure the failure is the same as the one that is fixed on `develop` ?

**Comment by harrisj** [comment]
Thanks, @jessieay. Rebased!

**Pull request merged**